coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/CalciteTrinoUDFMap.java

```diff
@@ -62,8 +62,7 @@ private CalciteTrinoUDFMap() {
     createUDFMapEntry(UDF_MAP, hiveToCalciteOp("hex"), 1, "to_hex");
     createUDFMapEntry(UDF_MAP, hiveToCalciteOp("unhex"), 1, "from_hex");
     createUDFMapEntry(UDF_MAP, hiveToCalciteOp("array_contains"), 2, "contains");
-    createUDFMapEntry(UDF_MAP, hiveToCalciteOp("regexp_extract"), 3, "regexp_extract",
-        "[{\"input\": 1}, {\"op\": \"hive_pattern_to_trino\", \"operands\":[{\"input\": 2}]}, {\"input\": 3}]", null);
```

Comment:
Curious what was the logic in `hive_pattern_to_trino` which we are missing. Should we add a hidden function `hive_pattern_to_trino` to Trino instead?

Reply:
> Curious what was the logic in `hive_pattern_to_trino` which we are missing. Should we add a hidden function `hive_pattern_to_trino` to Trino instead?

Here is the issue we proposed, we think that maybe the udf logic is in the internal version of linkedin presto:  
https://github.com/trinodb/trino/issues/12288

Reply:
Hi @findinpath @hackeryang @losipiuk, LinkedIn Trino has an internal udf `hive_pattern_to_trino` which converts the Hive regexp pattern to Trino regexp pattern. This function currently replaces consecutive backslashes with single backslash given Hive pattern uses double slash (\\\\) for escaping whereas trino requires single \\.

So I think it would be better to add this function to Trino.

Reply:
> Hi @findinpath @hackeryang @losipiuk, LinkedIn Trino has an internal udf `hive_pattern_to_trino` which converts the Hive regexp pattern to Trino regexp pattern. This function currently replaces consecutive backslashes with single backslash given Hive pattern uses double slash (\\) for escaping whereas trino requires single \.
> 
> So I think it would be better to add this function to Trino.

Thank you for your reply, @ljfgem . Can you please contribute this udf to the Trino community? We will appreciate it if the community can have this feature.